iceoryx_posh/include/iceoryx_posh/internal/popo/ports/interface_port_data.hpp

```diff
@@ -31,6 +31,7 @@ struct InterfacePortData : public BasePortData
     InterfacePortData() = default;
     InterfacePortData(const RuntimeName_t& runtimeName, const capro::Interfaces interface) noexcept;
 
+    /// @todo put condition variable data here
```

Comment:
Please no todos in the code.

Reply:
See [this](https://github.com/eclipse-iceoryx/iceoryx/pull/784#discussion_r633843244) comment.